CONTRIBUTING.md

```diff
@@ -52,6 +53,10 @@ From a separate terminal, start running a flutter app to connect to DevTools:
 - `flutter run`
 - copy the "Observatory debugger and profiler" uri printed in the command output, to connect to the app from DevTools later
 
+### **[Optional]** Enable DCM (Dart Code Metrics)
```

Comment:
What's the point to explain it in separate section and link here?
How about just explaining everything in 'Enable DCM (Dart Code Metrics)'?

Reply:
Considering that this section is optional, the idea was to only mention DCM here and describe everything at the bottom. You want to have everything described here?

Reply:
yes, there is no point to separate description

Reply:
There is only one now.